graphql_client/__init__.py

```diff
@@ -15,6 +15,9 @@
 import websocket
 
 
+WSS_HEADER = "Sec-WebSocket-Protocol: graphql-ws"
```

Comment:
This header is the sub-protocol header, and nothing to do with `wss` actually. Better off naming it something like `GQL_PROTO_HEADER` ?

https://tools.ietf.org/html/rfc6455#section-11.3.4
https://www.codeproject.com/Articles/716148/Using-Sec-Websocket-Protocol

Reply:
Very good point. I'll rename it to `GRAPHQL_SUBPROTOCOL_HEADER`